nservicebus/concept-overview.md

```diff
@@ -30,14 +30,14 @@ The payload of the message is also called the body. It travels between the endpo
 Additional information about a message is communicated over the transport in a collection of key value pairs. Message headers are similar to HTTP headers in the sense that they define how messages should be transmitted and processed. NServiceBus uses headers to implement some of its features. Users are also able to add [custom headers](/nservicebus/messaging/header-manipulation.md).
 
 
-### Endpoint
+### [Endpoint](/nservicebus/endpoint/)
 
-An Endpoint is a design-time concept that has a name and a collection of associated Message Handlers and Sagas. A definition of an Endpoint lives as a group of artifacts in a source code repository. An Endpoint (the binaries that result from building the endpoint's source code) can be deployed to a number of machines and environments. Each such deployment is a separate Endpoint Instance.
+An Endpoint is a logical entity that communicates with other Endpoints via messaging. Each Endpoint has an identifying name and contains a collection of Message Handlers and Sagas. An Endpoint can be deployed to a number of machines and environments. Each such deployment is a separate Endpoint Instance.
 
 
-### Endpoint Instance
+### [Endpoint Instance](/nservicebus/endpoint/)
 
-An Endpoint Instance is a run-time object that allows to interact with the bus. Endpoint Instances are able to send and receive messages. It runs associated Message Handlers and Sagas to process incoming messages. An Endpoint Instance has a single Input Queue (which can be shared with other instances of the same Endpoint in a scale-out scenario).
+An Endpoint Instance is a run-time object that allows interaction with the bus. Endpoint Instances are able to send, receive, and publish messages.. It runs associated Message Handlers and Sagas to process incoming messages. An Endpoint Instance has a single Input Queue which can be shared with other instances of the same Endpoint in a scale-out scenario.
```

Comment:
> An Endpoint Instance has a single Input Queue which can be shared with other instances of the same Endpoint in a scale-out scenario.

Is this true for MSMQ? True, an endpoint instance has a single input queue, but is it shared with other instances?